.eslintrc.js

```diff
@@ -64,12 +64,6 @@ module.exports = {
         'jsx-a11y/no-onchange': 'off',
       },
     },
-    {
-      files: ['src/legacy/core_plugins/console/**/*.{js,ts,tsx}'],
-      rules: {
-        'react-hooks/exhaustive-deps': 'off',
```

Comment:
Can  https://github.com/elastic/kibana/issues/49543 be closed once this PR is merged?

Reply:
Yep!